pom.xml

```diff
@@ -256,6 +256,14 @@
 
     <reporting>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>3.3.1</version>
+                <configuration>
+                    <skip>${javadoc.skip}</skip>
+                </configuration>
```

Comment:
Currently `javadoc.skip` defaults to `true`. Why do we want to skip the javadoc generation in `mvn site` and set this as default behavior?

Reply:
Removed this configuration.